### PR TITLE
Added addReferer method to confing reader

### PR DESCRIPTION
**Pull request by tiborb**
Can add referers even after the conf file has been loaded as stated in #115

### CODE REVIEW

File: php/src/Snowplow/RefererParser/Config/ConfigFileReaderTrait.php [hunk 1]
```diff
@@ -33,20 +35,37 @@ private function read()
...
+    
+    /**
+     * Add referer
+     * 
+     * @param string $domain 
+     * @param string $source
+     * @param string $medium
+     * @param array $parameters
+     */
+    public function addReferer($domain, $source, $medium, $parameters){
```
Comment (lstrojny) [inline php/src/Snowplow/RefererParser/Config/ConfigFileReaderTrait.php]: Brace should be in the next line

**Comment by lstrojny** [comment]
Can you extend `AbstractConfigReaderTest` to cover the case?

**Comment by tiborb** [comment]
Thanks, I will complete it

**Comment by tiborb** [comment]
@lstrojny are the changes ok so far?

File: php/src/Snowplow/RefererParser/Config/ConfigFileReaderTrait.php [hunk 2]
```diff
@@ -29,24 +31,44 @@ private function read()
...
-        foreach ($hash as $medium => $referers) {
-            foreach ($referers as $source => $referer) {
-                foreach ($referer['domains'] as $domain) {
-                    $this->referers[$domain] = [
-                        'source'     => $source,
-                        'medium'     => $medium,
-                        'parameters' => isset($referer['parameters']) ? $referer['parameters'] : [],
-                    ];
+        
+        if (!empty($hash)){
```
Comment (lstrojny) [inline php/src/Snowplow/RefererParser/Config/ConfigFileReaderTrait.php]: Superfluous as foreach will do nothing anyway if the array is empty

**Comment by lstrojny** [comment]
Could you remove the empty() check and squash the commits?

**Pull request closed without merging**